src/main/resources/init_db.sql

```diff
@@ -0,0 +1,8 @@
+use books_store_db;
+CREATE TABLE IF NOT EXISTS books
+(
+    id    bigint auto_increment

```

Comment:
Be consistent - if you use upper case so use it everywhere
 id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,